src/operators/rastervectorjoin.md

```diff
@@ -0,0 +1,77 @@
+# RasterVectorJoin
+
+The `RasterVectorJoin` operator allows combining a single vector input and multiple raster inputs.
+For each raster input, a new column is added to the collection from the vector input.
+The new column contains the value of the raster at the location of the vector feature.
+For features covering multiple pixels like `MultiPoints` or `MultiPolygons`, the value is calculated using an aggregation function selected by the user.
+The same is true if the temporal extend of a vector feature covers multiple raster time-steps.
+More details are described below.
+
+**Example**:
+You have a collection of aggricaltural fields (`Polygons`) and a collection of raster images containing the monthly NDVI value for each pixel.
+The `RasterVectorJoin` operator allows you to do this.
```

Comment:
Clarify: to do what?

Reply:
[ec1758e](https://github.com/geo-engine/docs/pull/47/commits/ec1758eeb071d2f3f612b9b54b9051f980628b87)